docs/WIRELESS.md

```diff
@@ -1,162 +1,134 @@
-
-For example, the following may be a reasonable session to initialize your modem using AT commands:
-
-```bash
-picocom -b 115200 /dev/ttyUSB2
-
-ati
-at!entercnd="A710"
-at+cpin?
-  READY
-at!custom="IPV6ENABLE",1
-at+cgdcont=1,"ipv4v6","vzwinternet"
-at!selrat=06
-at!usbcomp = 1,1,1009
-at!reset
-```
-
-#### 4. Establishing a data connection
-
-Once your modem reliably connects to your desired wireless provider, the final step is making sure that you can request a data connection. Data connection is layered on top of the basic GSM connectivity and requires you knowing a recommended APN and credentials that are needed to connect to it (both can be set dynamically by EVE's controller). Managing that data connection is the job of EVE's [wwan](../pkg/wwan) package and that is all that it does (e.g. it does NOT manage firmware or basic GSM connectivity). `wwan` package uses `uqmi` utility to talk to the QMI-enabled USB endpoint (typically `/dev/cdc-wdmX`) and goes through the following stages:
-
-```bash
-# wait for modem to register with the network
-uqmi --get-serving-system
-# start data connection
-uqmi --start-network --apn YOUR_APN --keep-client-id wds
-# wait for data connection to be established
-uqmi --get-data-status
-# wait for IP setting (addr, DNS, etc.) to be available
-uqmi --get-current-settings
-```
-
-In general, a single GSM modem can actually multiplex between different data networks (and thus provide multiple network interfaces) this is very rarely done in practice (and EVE certainly doesn't support it) but you need to keep in mind that each of these networks is distinguished by a separate Packet Data Handle (PDH) value. For example, your `uqmi --start-network ...` command will return a unique PDH handle back to you and if you ever want to reference that particular data connection you'll have to either use that value or use a catchall one `0xFFFFFFFF`.
-
-Another concept that you will encounter when looking at QMI/MBIM protocols is that of a Client ID (CID). Think of it as an HTTP token in REST APIs -- something that uniquely identifies a stateful connection with a given client. If you're issuing a series of QMI/MBIM commands as a transaction you want to keep client ID the same for all of them. Take a look at how [this script](https://github.com/freedesktop/libqmi/blob/master/utils/qmi-network.in) handles both PDH and CID.
-
-This idea of an token guaranteeing an active connection is actually a pretty important one. For example, another popular script for attaching to data connection in [MBIM mode](https://github.com/freedesktop/libmbim/blob/master/utils/mbim-network.in) goes out of its way to keep a single active connection between different invocations of the `mbimcli` utility (look for how it uses `--no-open` and `--no-close` options and caches transactions IDs so that the next call to `mbimcli` can pick it up and use it as an argument for `--no-open`)
-
-### Frequencies and antennas
```

Comment:
I moved this to wwan/README.md since I have a chapter there explaining how to check/add support for a new modem. Info on antennas is somewhat related so I wanted to have this together.

Reply:
OK, I moved it back to WIRELESS.md